pkg/minikube/cluster/cluster.go

```diff
@@ -359,11 +359,6 @@ func (m *MachineConfig) IsMinikubeISOCached() bool {
 }
 
 func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
-	if config.ShouldCacheMinikubeISO() {
```

Comment:
No need to remove it from here. This is not our code, probably better to leave it untouched.